### PR TITLE
Add instructions on how to get Auth0 .env values

### LINKED ISSUE #93: Add instructions in api README on how to get Auth0 fields

### PR DESCRIPTION
**Pull request by armiantos**
# Overview

New section can be found here [api-readme](https://github.com/UAlbertaCompEClub/compe-plus/tree/armi/api-readme/api#env-file)

# Changes

* Added blurred screenshot
* Added .env section in README

# Questions & Notes

# Issue tracking

Closes #93 

# Testing & Demo

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ We also have a few scripts to keep the code quality up to snuff. Run `npm run fo
 
 The API makes use of a number of services e.g., database, blog storage, email. See the [dependencies](#Dependencies) section for more on how to locally emulate these services.
 
+## .env file
```
Comment (calebschoepp) [inline api/README.md:19]: Nit: Maybe call this `Environment`?

**Review by calebschoepp — APPROVED** [review]
LGTM!

File: api/README.md (hunk 1 above)
Comment (armiantos) [reply api/README.md:19]: How about environment variables?

Comment (calebschoepp) [reply api/README.md]: Sure :D

**Pull request merged**